openml-lightgbm/lightgbm-provider/pom.xml

```diff
@@ -25,7 +25,7 @@
     <description>OpenML Microsoft LightGBM Machine Learning Model and Classifier provider</description>
 
     <properties>
-        <lightgbmlib.version>3.0.0</lightgbmlib.version>
+        <lightgbmlib.version>3.0.0-with_model_locale_fix_for_java</lightgbmlib.version>
```

Comment:
@shenggwang @paulojrp  now we point to the internal tag. :+1: ?

Reply:
It looks good to me